examples/simple_window.rs

```diff
@@ -35,8 +36,15 @@ struct Window {
 }
 
 impl wayland_window::Handler for Window {
-    fn configure(&mut self, _: &mut EventQueueHandle, _: wl_shell_surface::Resize, width: i32, height: i32) {
-        self.newsize = Some((width, height))
+    fn configure(&mut self, _: &mut EventQueueHandle, width: i32, height: i32) {
+        let w = std::cmp::max(width, 100);
+        let h = std::cmp::max(height, 100);
+        if width <= 0 || height <= 0 {
+            println!("WARNING: wayland_window::Handler::configure: Received invalid dimensions: \
```

Comment:
These are not invalid dimensions actually, they just indicate that the user doing the resize moved the cursor past the other side of the window than the side they grabbed. It's up to the client to decide how to act in this case. The most classic behavior being to ignore requests to be resized beyond a certain minimum size.